src/services/settings-service/settings-service.ts

```diff
@@ -22,7 +22,7 @@ export interface ISettingsService {
 
 @Injectable()
 export class SettingsService {
-    private theme: BehaviorSubject<string>;
+    theme: BehaviorSubject<string>;
```

Comment:
Would it help if we made this property readonly? Example: `readonly theme: BehaviorSubject<string>;`.

This way we make it clear this property should not be written to.

Reply:
Good point! Added next request